### PR TITLE
Feature/Log Model [EOSF-9]

### PR DESCRIPTION
**Pull request by pattisdr**
## Ticket
https://openscience.atlassian.net/browse/EOSF-9

# Purpose

Adds ember model for logs.

# Notes for Reviewers
-  Logs currently don't have their own route, but are they needed for pagination?
- Should params get their own model?  Log params are very different depending on the log type.
- Contributors not a part of the model because it will be removed from the log serializer in https://github.com/CenterForOpenScience/osf.io/pull/5487, to params

### Routes Added/Updated

No routes added.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -17,3 +17,4 @@ npm-debug.log
 testem.log
 
 config/*.yml
+.idea/
```
Comment (samchrisinger) [inline .gitignore]: I have no problem adding this, but you might want to look into setting up a global .gitignore: https://help.github.com/articles/ignoring-files/#create-a-global-gitignore

File: addon/models/node.js [hunk 2]
```diff
@@ -42,6 +42,7 @@ export default OsfModel.extend({
 
     root: DS.belongsTo('node', {
         inverse: null
-    })
-    //logs: DS.hasMany('node-logs'),
+    }),
+    //primaryInstitution: DS.belongsTo('institution'),
```
Comment (samchrisinger) [inline addon/models/node.js]: Probably just part of a merge, but as a note--
Henrique has added 'affilatedInstitutions' as the 'primaryInstitution' field will be going away soon: https://github.com/CenterForOpenScience/osf.io/pull/5450

Comment (pattisdr) [inline addon/models/node.js]: I had just corrected the spelling.

File: tests/dummy/app/styles/app.scss [hunk 3]
```diff
@@ -0,0 +1,7 @@
+table.logs, table.logs th, table.logs td {
```
Comment (samchrisinger) [inline tests/dummy/app/styles/app.scss]: These styles should live in a component-specific CSS file like here: https://github.com/CenterForOpenScience/ember-osf/tree/develop/tests/dummy/app/components/files-tree

https://github.com/CenterForOpenScience/ember-osf/blob/develop/.github/CONTRIBUTING.md#styling-components

File: .gitignore (hunk 1 above)
Comment (pattisdr) [inline .gitignore]: Thanks - I didn't know this was a thing.

**Comment by samchrisinger** [comment]
Overall, lgtm. Using a component is exactly the right strategy, 👍!  A couple small changes then good to merge.

**Pull request merged**